docs/docs/keymaps/input-processors/index.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Input Processor Overview
+sidebar_label: Overview
+---
+
+## Input Processors Overview
+
+"Input processors" are small pieces of functionality that process and optionally modify events generated from emulated and physical pointing devices. Processors can do things like scaling movement values to make them larger or smaller for detailed work, swapping the event types to turn movements into scroll events, or temporarily enabling an extra layer while the pointer is in use.
+
+## Usage
+
+For information on using input processors with a given pointing device, see [input processor usage](usage.md).
+
+# Available Processors
```

Comment:
```suggestion
## Available Processors
```
This is throwing off docusaurus, e.g. this header doesn't appear in the sidebar.

(I didn't change the level of "External Processors" assuming it should be a sibling to this one.)